### PR TITLE
Add welcome wagon to 2020 site

### PR DESCRIPTION
**Pull request by ericholscher**
This is 80% done. There's a couple sections I commented out, but I think it's better than nothing.

### CODE REVIEW

File: content/welcome-wagon/contents.lr [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+
+- *In person*: any staff member can find a CoC member
+- *Hotline*: `+1 (206) 258-5599 <tel:+12062585599>`_.
+- *Email*: conduct (at) pycascades.com
+- *Staff member issues*: Contact conference chairs directly:
+
+  * Eric: `+1 (757) 705 0532 <tel:+17577050532>`_.
+
+.. 
+    Contribute
```
Comment (nnja) [inline content/welcome-wagon/contents.lr:56]: This repo https://github.com/pycascades/welcome-wagon-2020 doesn't exist yet. Remove this section for the time being?

File: content/welcome-wagon/contents.lr [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+FAQs
+-----
+
+.. contents::
+   :local:
+   :backlinks: none
+
+.. 
+    **Where is everything?**
```
Comment (nnja) [inline content/welcome-wagon/contents.lr:98]: Update the venue information, and remove everything else?

File: content/welcome-wagon/contents.lr [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+    ~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Our venue is `Kane Hall on the campus of the University of Washington <https://goo.gl/maps/ZVVnBcmfmN22>`_.
+
+    The main conference takes place at room 130. Refreshments will be available during breaks at the Walker Ames Room (Room 225).
+
+    The `sprint <sprint.html>`_ is at The Collective Portland, `400 Dexter Avenue <https://goo.gl/maps/b1JY354TkvA2>`_.
+
+    **How should I get around?**
```
Comment (nnja) [inline content/welcome-wagon/contents.lr:107]: Remove this section?

File: content/welcome-wagon/contents.lr [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+
+**What's the weather like?**
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The weather in February in Portland is usually cold, but normally above
+freezing. You should expect grey days with light rain. In February,
+the temperature in Portland typically ranges from 41°F to 50°F (5°C to
+10°C) and is rarely below 32°F (0°C) or above 58°F (14°C).
+
+The weather forecast for the weekend is a high of 42°F (6°C) with a
```
Comment (nnja) [inline content/welcome-wagon/contents.lr:131]: Remove this paragraph

File: content/welcome-wagon/contents.lr [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+
+The most important part of this conference (and any conference) is the
+people you meet. Set a goal for yourself to meet a few, new people. Here
+are some tips:
+
+-  Find out who is attending the conference before you get there. Join
+   the `PyCascades Slack`_, follow
+   `PyCascades on Twitter <https://twitter.com/pycascades>`_,
+   and review the `list of
+   talks <https://www.pycascades.com/talks/>`_.
```
Comment (nnja) [inline content/welcome-wagon/contents.lr:226]: this 404s for now

**Review by nnja — CHANGES_REQUESTED** [review]
Few sections that can be left out, otherwise lgtm

**Review by nnja — APPROVED** [review]
apparently I don't know how comments in rst work. lgtm. 👍

**Comment by ericholscher** [comment]
@nnja yea, it's annoying that GH won't syntax highlight it because it's `.lr` -- nobody should know RST syntax by heart :D

File: content/welcome-wagon/contents.lr [hunk 6]
```diff
@@ -0,0 +1,287 @@
...
+ways you can make the most of the conference. The FAQs strive to answer
+questions before you even have them.
+
+
+Important Information
+----------------------
+
+- *Restrooms*: Located on each floor of the venue
+- *Refreshments*: Located downstairs in the Astoria room.
+- *Quiet room*: In the SunSet room on the first floor near the main auditorium.
```
Comment (EstiShay) [inline content/welcome-wagon/contents.lr]: `Sunset` not `SunSet`.

File: content/welcome-wagon/contents.lr [hunk 7]
```diff
@@ -0,0 +1,287 @@
...
+
+Code of Conduct
```
Comment (EstiShay) [inline content/welcome-wagon/contents.lr:41]: Maybe hold off on this part until Mariatta can check it? This last year's hotline number; will that be the same this year? And I'd like to be listed, but we don't have a temporary number set up for me yet and I wouldn't want it on the website until right before the conference.

Comment (ericholscher) [reply content/welcome-wagon/contents.lr:41]: Commented it out for now 👍

Comment (theacodes) [reply content/welcome-wagon/contents.lr:41]: I can reserve us a number from conducthotline.com whenever, and it can be inactive until we turn it on.

**Pull request merged**